src/amo/components/Category/index.js

```diff
@@ -67,11 +68,17 @@ export function mapStateToProps(state, ownProps) {
     { ...state.search.filters, page: parsePage(state.search.page) },
     { ...filters, page: queryParams.page },
   );
+
+  const hasFetched = state.categories.hasFetched;
```

Comment:
Could this be derived from the category data rather than needing to be passed as a prop?